server/src/controllers/github.controller.ts

```diff
@@ -99,6 +105,7 @@ export const repositories = async (
     }));
 
     res.json(repositories);
+    console.log("repo body",response.data);
```

Comment:
Try to use logger now

Also use it while development ,remove them while pushing to git

(unless its important)